Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -160,6 +168,34 @@ public SolverOffsetBehavior OffsetBehavior
             set => offsetBehavior = value;
         }
 
+        [SerializeField]
+        [Tooltip("Additional offset to apply towards the user.")]
+        private float forwardOffset = 0;
+
+        /// <summary>
+        /// Additional offset to apply towards the user.
+        /// </summary>
+        public float ForwardOffset
+        {
+            get { return forwardOffset; }
+            set { forwardOffset = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Additional degree offset to apply from the stated SafeZone. Ignored if Safe Zone is Atop Palm" + 
+        " Direction is clockwise on the left hand and anti-clockwise on the right hand.")]
+        private float safeZoneAngleOffset = 0;
+
+        /// <summary>
+        /// Additional degree offset to apply clockwise from the stated SafeZone.
+        /// Direction is clockwise on the left hand and anti-clockwise on the right hand.
+        /// </summary>
+        public float SafeZoneAngleOffset
+        {
+            get { return safeZoneAngleOffset; }
+            set { safeZoneAngleOffset = value; }
```

Comment:
Nit: Maybe we can use the

```
get => safeZoneAngleOffset;
set => safeZoneAngleOffset = value;
```

syntax? It is already used elsewhere in MRTK and is quite concise.